### PR TITLE
Add colors to seating chart seats by term

### PR DESCRIPTION
**Pull request by czhaocmu**
I can change the colors once we finalize them.

### CODE REVIEW

File: ap/seating/views.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from terms.models import Term
 from accounts.models import Trainee
 from .serializers import ChartSerializer, SeatSerializer, PartialSerializer, PartialFilter
-from accounts.serializers import TraineeSerializer, BasicUserSerializer
+from accounts.serializers import TraineeSerializer, BasicUserSerializer, TraineeRollSerializer
```
Comment (jontey) [inline ap/seating/views.py]: Is BasicUserSerializer still in use? if not we can delete it

Comment (czhaocmu) [inline ap/seating/views.py]: Nope, I removed it.

Comment (czhaocmu) [inline ap/seating/views.py]: termClass css is in seating.css now.

**Comment by jontey** [comment]
Good job @czhaocmu. Merging!

**Pull request merged**